app/controllers/api/v1/slack_users_controller.rb

```diff
@@ -2,9 +2,31 @@ module Api
   module V1
     class SlackUsersController < ApiController
       before_action :authenticate_py_bot!
+      before_action :set_slack_user, only: [:update]
 
       def create
-        # TODO: Placeholder for PyBot call to create new record
+        user = User.find_by(email: params[:slack_email])
+
+        ActiveRecord::Base.transaction do
+          slack_user = SlackUser.new slack_user_params
+          slack_user.user_id = user.id unless user.nil?
```

Comment:
@wimo7083 On the slack_user table it had a user_id specified so I was populating that based on the user that returns from the e-mail address query.

Do have a concern with this that maybe the slack user has the e-mail specified for a normal user of the application then the slack_user will be related to a user that isn't theirs. I am assuming that the slack users are created through the py bot application exclusively? Maybe we don't want to populate the user_id right on create, but have a different action with pybot to be able to link a user and a slack_user? I'm not sure.

Just was a little concerned with setting the user ID and wanted to see if I should keep it or take it out?

Reply:
TODO: later- if we don't find a user_id matching slack log this somehow.